.gitattributes

```diff
@@ -0,0 +1 @@
+*.strings diff
```

Comment:
What is the reason for this addition?

Reply:
I wanted to present .strings files as textual files, not as binday, so I we could review/compare them in SourceTree, Fork, etc..